src/components/projectDetail/AddressAddModal.tsx

```
@@ -0,0 +1,110 @@
+      className="rounded-2xl p-6 shadow-xl"
+    >
+      <div className="mb-6 flex items-center justify-between">
+        <h4 className="text-xl font-bold">새 배송지 추가</h4>
+        <button onClick={onClose} className="rounded-full p-1 hover:bg-gray-100">
+          <X className="h-6 w-6" />
+        </button>
+      </div>
+      <div className="space-y-4">
+        <div>
+          <label className="block text-sm font-medium text-sub-gray mb-2">주소</label>
+          <div className="flex space-x-2">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)